src/partcad/cae.py

```
@@ -0,0 +1,467 @@
+            entry["load"] = amount
+            matched.add("load:" + name)
+
+        if entry is not None:
+            entry.setdefault("fix", False)
+            entry.setdefault("load", 0.0)
+            assigned.append(entry)
+
+    unmatched = []
+    for key, declared in (("fix", config.fixtures), ("load", config.loads)):
+        for name in declared:
+            if "%s:%s" % (key, name) in matched:
```

Ruff (0.16.3)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)